packages/aml-checklist/src/components/ui/modal/address-information/form-schema/validation-schema.ts

```diff
@@ -0,0 +1,21 @@
+import * as Yup from 'yup'
+import errorMessages from '@/constants/error-messages'
+import { isValidUploadForm } from '@/utils/validates'
+
+/* istanbul ignore next */
+const validationSchema = Yup.object().shape({
+  metadata: Yup.object().shape({
+    primaryAddress: Yup.object().shape({
+      documentImage: Yup.string()
+        .required(errorMessages.FIELD_REQUIRED)
+        .test('isValidBase64', 'Wrong file type', value => isValidUploadForm(value)),
```

Comment:
Please add a new constant for 'wrong file type' error message to `@/constants/error-messages` and use from there